library/styleguideComponents/ComponentsList/index.js

```diff
@@ -37,6 +39,14 @@ export default class ComponentsList extends Component {
                 onSelect={this.handleOnSelect}>
 
                 <Sections selectedItem={selectedItem} items={items} searchTerm={searchTerm} />
+
+                <VerticalSection>
+                    <VerticalItem
+                        name="Experiences"
+                        selectedItem={selectedItem}
```

Comment:
remove this since VerticalItem does not receive selectedItem and fix the @AlejandroYanes one